docker/mysql/db/db_upgrade/portal/Update_RMC_DB_5372-5410.sql

```diff
@@ -4,7 +4,7 @@ ALTER TABLE `settings` CHANGE `setting` `setting` VARCHAR(40) CHARACTER SET utf8
 ALTER TABLE `settings` CHANGE `data` `data` TEXT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL;
 INSERT INTO `settings` (`setting`, `data`) VALUES ('siteType', 'standard');
 INSERT INTO `settings` (`setting`, `data`) VALUES ('national-linkedPrimary', ''), ('national-linkedSubordinateList', '');
-UPDATE `settings` SET `setting` = 'subHeading' WHERE `settings`.`setting` = 'subHeading';
+UPDATE `settings` SET `setting` = 'subHeading' WHERE `settings`.`setting` = 'subheading';
```

Comment:
Wouldn't you leave old values alone and just update it with the new db changes?

Reply:
These were changed in a previous commit, I'm changing them back and taking care of it in the two newest files.